docs/running.md

```diff
@@ -469,21 +469,21 @@ gVUBRpGWKyQvYeoOY3OJROSyogmA3ys=
 
 The simplest transaction we can do is to transfer tokens from one account to another.
```

Comment:
The `balance` in Genesis in this file hasn't been updated.

Reply:
Good catch, thanks!